Core/scripts/dirac-cert-convert.sh

```diff
@@ -17,56 +17,56 @@ function usage {
   exit 1
 }
 
-if [ $# = 0 ]; then
+if [[ $# = 0 ]]; then
   echo User Certificate P12 file is not given.
   usage
 fi
 
-export OPENSSL_CONF=$HOME/.globus
-GLOBUS=$HOME/.globus
+export OPENSSL_CONF="${HOME}"/.globus
+GLOBUS="${HOME}"/.globus
 USERCERT_P12_ORIG=$1
-USERCERT_P12=$GLOBUS/$(basename "$USERCERT_P12_ORIG")
-USERCERT_PEM=$GLOBUS/usercert.pem
-USERKEY_PEM=$GLOBUS/userkey.pem
+USERCERT_P12="${GLOBUS}"/$(basename "${USERCERT_P12}_ORIG")
```

Comment:
Is this a typo?
```suggestion
USERCERT_P12="${GLOBUS}"/$(basename "${USERCERT_P12_ORIG}")
```

Reply:
This applies to several instances of `USERCERT_P12_ORIG`

Reply:
Ouf, thanks, that was a bad replacement. Fixed.